ext/bcmath/tests/gh16236.phpt

```diff
@@ -0,0 +1,17 @@
+--TEST--
+GH-16236 Segmentation fault (access null pointer) in ext/bcmath/libbcmath/src/rmzero.c:50
+--EXTENSIONS--
+bcmath
+--FILE--
+<?php
+/**
+ * The existing bcpow() specification returns 0 for negative powers.
+ * This is mathematically incorrect and will need to be changed to raise an error at some point.
+ * This test is to ensure the existing specifications until the specifications are changed.
+ */
+bcpow('0', '-2');
```

Comment:
Please add a stub of this to https://wiki.php.net/rfc/deprecations_php_8_5 so it is not forgotten, or follow-up PR to make this emit an E_WARNING or Throwing on master.